debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/TypeRegistry.java

```diff
@@ -405,6 +405,9 @@ private PostgresType loadType(Connection connection, PreparedStatement statement
             while (rs.next()) {
                 PostgresType result = createTypeBuilderFromResultSet(connection, rs, typeInfo, sqlTypeMapper).build();
                 addType(result);
+                if (!connection.getAutoCommit()) {
```

Comment:
So where is this transaction started? For the sake of symmetry, I  think BEGIN/COMMIT should be in one and the same method.

Reply:
@gunnarmorling Don't forget that JDBC interface does not offer explicit `begin`. This is started when a first statement is executed.